src/pages/public/ForForeldre/components/ForForeldre.tsx

```diff
@@ -1,7 +1,30 @@
 import React from "react";
+import TextPictureCard from "../../../../components/Pages/TextPictureCard";
+import getContent from "../../../../api/ForForeldre";
```

Comment:
These imports can get quite messy when traversing back multiple directories.

A better approach (when more than two ../..) would be this:
![image](https://user-images.githubusercontent.com/46197518/112278263-ee80fd00-8c82-11eb-89bb-173708dbfe84.png)

Though, this requires registering "src" as the baseUrl in tsconfig.json.
Have a look at the official documentation for this:
[docs/Absolute-imports](https://create-react-app.dev/docs/importing-a-component/#absolute-imports)